casbin/core_enforcer.py

```diff
@@ -63,14 +64,16 @@ def init_with_model_and_adapter(self, m, adapter=None):
             self.load_policy()
 
     def _initialize(self):
-        self.rm = default_role_manager.RoleManager(10)
+        self.rm = dict()
```

Comment:
use the variable name of Go

Reply:
rmMap is meant to map in GoLang,Which named dict in Python,should I change it to rmDict?